app/move/middleware.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 
     try {
       res.locals.move = await moveService.getById(moveId)
+      res.locals.moveId = moveId
```

Comment:
What's the benefit of setting this as a separate variable?

Reply:
None really. Other than being able to use it as is. I think I probably did this before rewriting elsewhere to have getMove.

Reply:
Maybe lets remove this then if its not really needed. It's used in various views/controllers already. I'd say unless we're changing it everywhere there's not much point adding it for now.

Reply:
Done

Reply:
This still seems in there